### PR TITLE
blog: Announcing free Single Sign On for GraphQL Hive

### PR DESCRIPTION
**Pull request by n1ru4l**

### CODE REVIEW

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+> GraphQL Hive is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics. Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
+
+GraphQL Hive gives you the flexibility of choosing your existing identity provider of choice for authenticating users, such as Google Workspaces, Okta, Auth0, and many more.
+
+Users logging into GraphQL Hive via an OIDC provider will be automatically added and limited to interacting within your organization.
+
+You can start using OIDC SSO on the GraphQL Hive Cloud version for any type of organization at no additional cost starting today.
+
+Offering this feature for free is a big deal for us.
+We at The Guild believe that Single Sign On is a core security requirement and not a luxury feature that you have to buy in.
```
Comment (Urigo) [inline website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:24]: Should we link to the SSO wall of shame?

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Today we are excited to announce that due to popular demand we now support connecting a GraphQL Hive Organization to an OAuth Open ID Connect Provider (OIDC) for Single Sign On (SSO).
+
+> GraphQL Hive is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics. Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
+
+GraphQL Hive gives you the flexibility of choosing your existing identity provider of choice for authenticating users, such as Google Workspaces, Okta, Auth0, and many more.
+
+Users logging into GraphQL Hive via an OIDC provider will be automatically added and limited to interacting within your organization.
+
+You can start using OIDC SSO on the GraphQL Hive Cloud version for any type of organization at no additional cost starting today.
```
Comment (Urigo) [inline website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:21]: Should we mention and thank Supertokens or it has nothing to do with them?
If it does, maybe let's write it in a way that they would want to tweet about it themselves as well?

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+title: Announcing free Single Sign On for GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: laurin
+date: 2022-11-04
+description: Link your GraphQL Hive Organization to your OAuth OIDC provider of choice.
+image: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/thumbnail.png
+---
+
+[Last month](/blog/announcing-self-hosted-graphql-hive) we announced self-hosted GraphQL Hive.
```
[the remark this answers is not in the document]
Comment (Urigo) [reply website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:11]: I would start with the announcement, the feature and then later I would say it is available both for the hosted solution and the self hosted solution that was announced last month (with the link to the blog post)

**Review by Urigo — CHANGES_REQUESTED** [review]

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx (hunk 1 above)
Comment (kamilkisiela) [reply website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:22]: I think we can!

**Comment by n1ru4l** [comment]
@Urigo let me know if it is better now, also feel free to edit it yourself!

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+authors: laurin
+date: 2022-11-04
+description: Link your GraphQL Hive Organization to your OAuth OIDC provider of choice.
+image: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/thumbnail.png
+---
+
+Today we are excited to announce that due to popular demand we now support connecting a GraphQL Hive Organization to an OAuth Open ID Connect Provider (OIDC) for Single Sign On (SSO).
+
+> GraphQL Hive is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics. Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
```
Comment (Urigo) [inline website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:13]:
```suggestion
> [GraphQL Hive](https://graphql-hive.com/) is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics. Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
```

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+GraphQL Hive gives you the flexibility of choosing your existing identity provider of choice for authenticating users, such as Google Workspaces, Okta, Auth0, and many more.
+
+Users logging into GraphQL Hive via an OIDC provider will be automatically added and limited to interacting within your organization.
+
+You can start using OIDC SSO on the GraphQL Hive Cloud version for any type of organization at no additional cost starting today.
+
+Offering this feature for free is a big deal for us.
+We at The Guild believe that Single Sign On is a core security requirement and not a luxury feature that you have to buy in.
+
+You can learn more about the broken relationship between SaaS vendors and SSO on the [SSO Wall of Shame](https://sso.tax/).
```
Comment (Urigo) [inline website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:24]: @n1ru4l also after we release this let's send a PR there: https://github.com/robchahin/sso-wall-of-shame

File: website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+In addition, OAuth Open ID Connect login is not only limited to the GraphQL Hive Cloud version, but is also available on self-hosted GraphQL Hive [which we announced Last month](/blog/announcing-self-hosted-graphql-hive).
+
+Huge thanks go out to the incredible helpful team at [SuperTokens](https://supertokens.com/) that helped us a lot with questions and feedback on shipping this feature.
+GraphQL Hive uses the SuperTokens Open Source project for user authentication.
```
Comment (Urigo) [inline website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx:29]:
```suggestion
GraphQL Hive is [fully open source](http://github.com/kamilkisiela/graphql-hive) and also uses the SuperTokens Open Source project for user authentication.
```

**Review by Urigo — CHANGES_REQUESTED** [review]

**Review by Urigo — APPROVED** [review]

**Comment by n1ru4l** [comment]
merging tomorrow! https://github.com/kamilkisiela/graphql-hive/pull/524

**Review by enisdenjo — APPROVED** [review]

**Pull request merged**